src/components/ec-container/types.ts

```diff
@@ -1,3 +1,4 @@
 export interface ContainerProps {
   isCollapsable?: boolean,
+  showMobileLayout?: boolean,
```

Comment:
we name such props `isResponsive` e.g. smart-table

Reply:
It's `true` by default and we need to set this prop based on flag value in EBO

Reply:
yeap, however we will need to set `isResponsive` to false for now or it will break EBO UI, should we do a release now to amend it or wait and add it in [this ticket](https://fxsolutions.atlassian.net/browse/CHUX-501)?

Reply:
Ok, let it be false by default.